src/styles/blocks/catalog.scss

```diff
@@ -13,10 +12,19 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    transition: all 0.3s;
```

Comment:
don't use all with transition, it affects in bad way on website performance
specify only needed properties